mackerel-plugin-fluentd/lib/fluentd.go

```diff
@@ -17,15 +17,14 @@ import (
 
 var logger = logging.GetLogger("metrics.plugin.fluentd")
 
-var prefix = "fluentd"
-
 func metricName(names ...string) string {
-	return prefix + "." + strings.Join(names, ".")
+	return strings.Join(names, ".")
 }
 
-// FluentdMetrics plugin for fluentd
-type FluentdMetrics struct {
+// FluentdPlugin mackerel plugin for Fluentd
+type FluentdPlugin struct {
```

Comment:
Please alias FluentdPlugin as FluentdMetrics for backward compatibility.

```go
type FluentdMetrics = FluentdPlugin
```

Reply:
Fixed. 6f541bf